qoqo_calculator/src/calculator_complex.rs

```diff
@@ -690,7 +690,7 @@ mod tests {
     #[test]
     fn display() {
         let x = CalculatorComplex::new(-3, 2);
-        let x_formatted = format!("{}", x);
```

Comment:
The corresponding trait (Display) hasn't been updated to the next printing syntax

Reply:
(line 240)